api/python/provisioner/api.py

```diff
@@ -413,6 +413,24 @@ def get_swupgrade_info(release: str = None, iso_path: str = None,
     return _api_call('get_swupgrade_info', release=release, iso_path=iso_path,
                      nowait=nowait)
 
+def get_iso_version(release: str = None, nowait=False):
+    """
+    Return latest available ISO version.
+
+    Parameters
+    ----------
+    release:
+        (optional) SW Upgrade repository release version
+    nowait: bool
+        (optional) Run asynchronously. Default: False
+
+    Returns
+    -------
+    None
```

Comment:
I think we need to specify what returns this function because `get_iso_version` commands return upgrade version (string) or None. So the returning type is `Union[str, None]`. Also, please add the `Raises` section here since `get_iso_version` also can raise a `ProvisionerError` if `upgrade_ver < release_ver`

Reply:
done